src/App.jsx

```diff
@@ -11,15 +11,26 @@ import AuthenticatedRoute from './components/Routes/AuthenticatedRoute';
 import UnauthenticatedRoute from './components/Routes/UnauthenticatedRoute';
 import { AppContext } from './AppContext';
 
-import { mockedAuthorsList, mockedCoursesList } from './constants';
 import './App.css';
+import { loadAuthors, loadCourses } from './services';
+import { useDispatch } from 'react-redux';
+import { saveCourses } from './store/courses/actionCreators';
+import { saveAuthors } from './store/authors/actionCreators';
 
 function App() {
 	const [isAuthenticated, setIsAuthenticated] = useState(
-		localStorage.getItem('authKey')
+		localStorage.getItem('token')
 	);
-	const [coursesList, setCoursesList] = useState(mockedCoursesList);
-	const [authorsList, setAuthorsList] = useState(mockedAuthorsList);
+	const dispatch = useDispatch();
+
+	useEffect(() => {
```

Comment:
Right. I moved this because every time I created course, the Courses component shows only courses from backend, not from store. But I moved it back to Courses :)